lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -164,10 +161,7 @@ async def _sync_subscribers(self) -> bool:
                 self._loop,
             )
         except grpc.RpcError as err:
-            logging.error(
-                "Sync subscribers request error! [%s] %s", err.code(),
-                err.details(),
-            )
+            _log_grpc_error(err, "Sync subscribers")
```

Comment:
If we want to pass the function name, we should use the exact same naming pattern: `_sync_subscribers` in this case. Is there a way to get this name programmatically? something like this? https://stackoverflow.com/questions/251464/how-to-get-a-function-name-as-a-string

Reply:
Good idea. Changed the arguments accordingly.

Reply:
Actually do you think we could make it so that the caller doesn't even have to pass in its name manually? I can imagine we might start seeing some bad copy pastes leading to confusion :D

Reply:
Sure :D Amended another suggestion. What do you think?

Reply:
Looking good! added another comment